runtime/doc/lua.txt

```diff
@@ -1362,8 +1362,12 @@ notify({msg}, {level}, {opts})                                  *vim.notify()*
                              the user.
                     {level}  (number|nil) One of the values from
                              |vim.log.levels|.
-                    {opts}   (table|nil) Optional parameters. Unused by
-                             default.
+                    {opts}   (table|nil) Optional parameters.
+                             • once (boolean, string or nil) Display a
```

Comment:
@dundargoc When I format it as `boolean|string|nil`, the `|string|` is parsed as a tag.